pkg/scyllaclient/config.go

```diff
@@ -1,11 +1,11 @@
 package scyllaclient
 
 import (
+	"fmt"
 	"net/http"
 	"time"
 
-	"github.com/pkg/errors"
-	"go.uber.org/multierr"
+	"k8s.io/apimachinery/pkg/util/errors"
```

Comment:
nit: we usually  call this `apierrors` to avoid conflict with stdlib.

Reply:
done